frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -507,6 +507,8 @@ export const OperatorHubTileView: React.FC<OperatorHubTileViewProps> = (props) =
     validSubscriptionFilters: t('olm~Valid subscription'),
   };
 
+  const showMoreText = t('olm~Show more');
```

Comment:
@tvu20 curious about one thing. Do we have the hidden count here? The PatternFly FilterSidePanelCategory component will render the following:
``{showText || `Show ${hiddenCount} more`}``
Instead of 'Show more' I'm wondering if we can calculate the hidden count and pass in 'Show {{count]] more'?

Reply:
There doesn't seem to be any documentation on the Patternfly website on how many elements are hidden - I tried calculating it by subtracting the total elements from the amount shown but the amount of elements visible above the show more button doesn't seem to be consistent. I can look into this more tomorrow

Reply:
@nicolethoen any tips on how to accomplish displaying the # count when localizing the content in the `FilterSidePanelCategory` component?

Reply:
@jcaianirh @tvu20 
So the `FilterSidePanelCategory` component lives in the `react-catalog-view-extension` which has not necessarily had the same i18n treatment that the supported pf packages received.

The `hiddenCount` is calculated in the component, so not something you can read, but it is calculated the following way:
```
const childrenArray = childrenToArray(children);
const itemCount = childrenArray.length;
const hiddenCount = itemCount - maxShowCount;
```
So you could possibly calculate that yourself?